utils/ticker.go

```diff
@@ -18,8 +18,8 @@ func SlowLogger(ctx context.Context, msg, fieldName, fieldVal string, logger log
 		for {
 			select {
 			case <-slowTicker.C:
-				elapsed := time.Since(startTime).String()
-				logger.CWarnw(ctx, msg, fieldName, fieldVal, "time_elapsed", elapsed)
+				secondsElapsed := time.Since(startTime).Seconds()
+				logger.CWarnw(ctx, msg, fieldName, fieldVal, "seconds_elapsed", secondsElapsed)
```

Comment:
Using `Seconds()` here means that `2.0001` and `2.002` are treated identically in terms of log deduplication, which is totally fine as this function really only cares about second-level granularity.